### PR TITLE
Fix: swap input_amplitude and target_amplitude in JukeboxLoss.forward

### LINKED ISSUE #8820: [Bug] `JukeboxLoss.forward`: `input_amplitude` and `target_amplitude` variable names are swapped
## Describe the bug

In `monai/losses/spectral_loss.py`, `JukeboxLoss.forward()` has the variable names `input_amplitude` and `target_amplitude` **swapped**:

```python
def forward(self, input: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
    input_amplitude = self._get_fft_amplitude(target)   # ← BUG: should be _get_fft_amplitude(input)
    target_amplitude = self._get_fft_amplitude(input)   # ← BUG: should be _get_fft_amplitude(target)

    loss = F.mse_loss(target_amplitude, input_amplitude, reduction="none")
```

## Impact

While this does **not** affect the numerical output of the loss (since MSE is symmetric: `mse(a, b) == mse(b, a)`), the naming is **semantically inverted** which:

1. Makes the code **misleading and confusing** to anyone reading, extending, or debugging it
2. Creates a **silent maintenance hazard** — if anyone adds asymmetric processing (e.g. normalization, weighting, gradient masking) based on input vs. target, the swap will cause subtle, hard-to-debug training errors
3. Violates the principle of least surprise for the standard `forward(input, target)` contract that all PyTorch loss functions follow

## Root Cause

In `forward(self, input, target)`:
- `input_amplitude` is computed from `target` (wrong)
- `target_amplitude` is computed from `input` (wrong)

## Expected behavior

```python
def forward(self, input: torch.Tensor, target: torch.Tensor) -> torch.Tensor:
    input_amplitude = self._get_fft_amplitude(input)    # correct
    target_amplitude = self._get_fft_amplitude(target)  # correct
    loss = F.mse_loss(target_amplitude, input_amplitude, reduction="none")
```

## Location

`monai/losses/spectral_loss.py` lines 55-56

Affects all MONAI versions containing `JukeboxLoss`. Reproducible on `dev` branch as of 2026-04-11.

### PR DESCRIPTION
**Pull request by Zeesejo**
Fixes #8820

### Description

In `JukeboxLoss.forward()`, the variable names `input_amplitude` and `target_amplitude` were swapped:
- `input_amplitude` was computed from `target` (should be from `input`)
- `target_amplitude` was computed from `input` (should be from `target`)

This fix corrects the assignments to match semantic meaning and the standard `forward(input, target)` PyTorch convention.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [x] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by Zeesejo** [comment]
**Update**: This PR now also includes a second commit (`4417d79`) that fixes a docstring bug in `SSIMLoss.forward()` (tracked in #8822).

The `forward()` docstring examples showed `print(1-SSIMLoss()(x,y))`, but since `SSIMLoss` already computes `1 - ssim_value` internally, those examples would actually return the raw SSIM value (not the loss). This fix removes the incorrect `1-` prefix from all three docstring print examples.

**Review by ericspod — APPROVED** [review]
Hi @Zeesejo thanks for this change, I think it's fine and of course the behaviour hasn't changed since `mse_loss` was symmetric. I don't think the Copilot comments need to be addressed.
